src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -146,20 +146,50 @@ partial class BoundForEachStatement : IForEachLoopStatement
 
     partial class BoundSwitchStatement : ISwitchStatement
     {
+        private static readonly ConditionalWeakTable<BoundSwitchStatement, object> s_switchSectionsMappings =
```

Comment:
@CyrusNajmabadi Just curious, any other benefits of using StrongBox besides the actual type is explicit declared?